.claude/rules/infra/terraform.md

```
@@ -0,0 +1,16 @@
+---
+paths:
+  - infra/**
+---
+
+# Infrastructure (Terraform)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)